engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -97,8 +101,11 @@ def _jsonlines_output_formatter(token: Token, first_token: bool,
             final_dict["details"] = {
                 "finish_reason": details.get("finish_reason", None),
                 "generated_tokens": details.get("generated_tokens", None),
-                "inputs": details.get("inputs", None)
+                "inputs": details.get("inputs", None),
             }
+            if "prompt_tokens_details" in details:
```

Comment:
Is this following TGI streaming use case?

Reply:
Tried TGI container 2.0 

```
data:{"error":"Input validation error: `decoder_input_details` == true is not supported when streaming tokens","error_type":"validation"}
```

Reply:
ok, then we can define the way we like